packages/form-js-viewer/src/util/index.js

```diff
@@ -124,29 +130,33 @@ export function getSchemaVariables(schema, expressionLanguage = new FeelExpressi
       variables = [ ...variables, key ];
     }
 
-    if (valuesKey) {
-      variables = [ ...variables, valuesKey ];
-    }
+    if (direction === 'input' || direction === 'both') {
```

Comment:
In case we decide to drop `both`

Reply:
cf. https://github.com/bpmn-io/form-js/pull/611/files#r1266803208 , this would also simplify the checks here as we would only need 
```suggestion
    if (inputs) {
```